automation/Extension/webext-instrumentation/package.json

```
@@ -64,7 +64,7 @@
     "@types/firefox-webext-browser": "^63.0.0",
```

Could we bump this to [latest version](https://www.npmjs.com/package/@types/firefox-webext-browser) 
```suggestion
    "@types/firefox-webext-browser": "^70.0.1",
```

---

Done!